qm-start: add page (#8704)
tldr-pages/tldr: pages/linux/qm-start.md

# qm start

> > Start a virtual machine on Qemu/KVM Virtual Machine Manager.

Reviewer: ```suggestion
> Start a virtual machine on QEMU/KVM Virtual Machine Manager.
```

> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.

- Start a specific virtual machine:

`qm start {{100}}`

> - Specify the Qemu machine type (i.e. the CPU to emulate):

Reviewer: ```suggestion
- Specify the QEMU machine type (i.e. the CPU to emulate):
```

`qm start {{100}} --machine {{q35}}`

> - Timeout if the machine doesn't start within 60 seconds:

Reviewer 2: what about?
```suggestion
- Start a specific virtual machine with a timeout in seconds:
```

Reviewer 3: > what about?

Just a small addition. Since it is a description with an example, I think mentioning 60 would be better.

```suggestion
- Start a specific virtual machine with a timeout in 60 seconds:
```

Author: It does sound a little more fluent in reading, I agree.
